### DISCUSSION TITLE
「PWN」【香山杯2023】Writeup WP 复现 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#11] [Announcements]
# 「PWN」【香山杯2023】Writeup WP 复现 🐱 ネコのメモ帳

两题简单 PWN，单纯更新一下，记录一下 Python 调试

https://ouo.sh/reproducing/2023/10/17/%e9%a6%99%e5%b1%b1%e6%9d%af2023/

**Comment by MuelNova** [comment]
其实似乎可以使用 gdb 指令
```sh
set breakpoint pending on
```
之后直接打断点
